variables.tf

```diff
@@ -206,3 +206,9 @@ variable "docker_registry_auth_secret_name" {
   type        = "string"
   default     = ""
 }
+
+variable "prometheus_enabled" {
+  description = "Enable to expose the Prometheus endpoint. Also enables Prometheus metrics scraping from k8s pods - true or false."
+  default     = "false"
```

Comment:
can you use `type = bool` and set `default = false` without quotes, new terraform versions shows warning with old syntax.

Reply:
Thanks, will do